sentry/src/main/java/io/sentry/SentryWrapper.java

```diff
@@ -27,16 +27,18 @@ public final class SentryWrapper {
    * @return the wrapped {@link Callable}
    * @param <U> - the result type of the {@link Callable}
    */
+  @SuppressWarnings("deprecation")
   public static <U> Callable<U> wrapCallable(final @NotNull Callable<U> callable) {
-    final IHub newHub = Sentry.getCurrentHub().clone();
+    // TODO replace with forking
+    final IScopes newHub = Sentry.getCurrentScopes().clone();
```

Comment:
Should we call this `newScopes`?

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366